app/cmd/plugin_search.go

```diff
@@ -72,19 +72,70 @@ func searchPlugins(plugins *client.AvailablePluginList, keyword string) []client
 	return result
 }
 
+func matchPluginsData(plugins []client.AvailablePlugin) (result []client.CenterPlugin) {
+	result = make([]client.CenterPlugin, 0)
+	jclient := &client.UpdateCenterManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(jclient.JenkinsCore))
+	site, err := jclient.GetSite()
+	if err != nil {
+		return
+	}
+	pluginJclient := &client.PluginManager{
+		JenkinsCore: client.JenkinsCore{
+			RoundTripper: pluginSearchOption.RoundTripper,
+		},
+	}
+	getCurrentJenkinsAndClient(&(pluginJclient.JenkinsCore))
+	plu, err := pluginJclient.GetPlugins()
+	if err != nil {
+		return
+	}
+
+	for _, plugin := range plugins {
+		for _, updatePlugin := range site.UpdatePlugins {
+			if plugin.Name == updatePlugin.Name {
+				result = append(result, updatePlugin)
+				break
+			}
+		}
+		for _, availablePlugin := range site.AvailablesPlugins {
+			if plugin.Name == availablePlugin.Name {
+				result = append(result, availablePlugin)
+				break
+			}
+		}
+		for _, pl := range plu.Plugins {
+			if plugin.Name == pl.ShortName {
+				s := client.CenterPlugin{}
+				s.CompatibleWithInstalledVersion = false
+				s.Name = pl.ShortName
+				s.Installed.Active = true
+				s.Installed.Version = pl.Version
+				s.Title = plugin.Title
+				result = append(result, s)
+				break
+			}
+		}
+	}
+	return
+}
+
 // Output output the data into buffer
 func (o *PluginSearchOption) Output(obj interface{}) (data []byte, err error) {
 	if data, err = o.OutputOption.Output(obj); err != nil {
-		pluginList := obj.([]client.AvailablePlugin)
+		pluginList := obj.([]client.CenterPlugin)
 		buf := new(bytes.Buffer)
 
 		if len(pluginList) != 0 {
 			table := util.CreateTable(buf)
-			table.AddRow("number", "name", "installed", "title")
+			table.AddRow("number", "name", "installed", "version", "usedVersion", "title")
```

Comment:
Please consider a better name instead of `usedVersion`.

Reply:
`updateVersion` Is it okay?

Reply:
How about `version` and `installedVersion`?

Reply:
I think it's ok